jaeger_client/reporter.py

```diff
@@ -75,12 +74,14 @@ def report_span(self, span):
 
 class Reporter(NullReporter):
     """Receives completed spans from Tracer and submits them out of process."""
-    def __init__(self, channel, queue_capacity=100, batch_size=10,
+    def __init__(self, channel, sender=None, queue_capacity=100, batch_size=10,
```

Comment:
shouldn't the new `sender` parameter be the last parameter since users could be calling this function without explicitly naming(?) the parameters?

Reply:
and add a TODO to fix that in the next major rev.

Reply:
@black-adder you're right! Should have been fixed with also the todo @yurishkuro asked for.
Is it maybe a good thing open the issue for that and assign it to the v4 milestone (if it exists)?

 Let me know is anything else is necessary :)